.github/workflows/check.yml

```diff
@@ -133,8 +133,6 @@ jobs:
           npm run build:benchmarks
 
       - name: Check
-        # only run on newest node version
```

Comment:
Why removing this? Does it make sense to run this on all node verions?

Reply:
No, it's better to run everything on all Node.js versions.
Who knows that something will break :/